dev.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: turbograft
+up:
+- ruby: 2.3.1
+- bundler
+commands:
+  test: bundle exec rake test
```

Comment:
In Shopify/shopify this is done by `bin/rake` instead of `bundle exec rake`: https://github.com/Shopify/shopify/blob/master/dev.yml#L53

Does that do the same thing?